src/types.ts

```diff
@@ -59,12 +59,17 @@ export class types {
     return scalarType()
   }
 
+  static or<T, U>(_t: T, _u: U): T | U {
+    return scalarType()
+  }
+
```

Comment:
Could you support `or` types up to 12 types with parameter overrides?

Like this: https://github.com/reduxjs/reselect/blob/master/src/index.d.ts#L22

Reply:
Does it really needed? When you need more than two types it could be written like this `types.or(types.string, types.or(types.number, ...))`